### PR TITLE
Add CVE-2020-26297 to mdBook <= 0.4.4

### PR DESCRIPTION
**Pull request by pietroalbini**
This PR adds https://groups.google.com/g/rustlang-security-announcements/c/3-sO6of29O0 to RustSec.

### CODE REVIEW

File: crates/mdbook/RUSTSEC-0000-0000.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+[versions]
+patched = [">= 0.4.5"]
+unaffected = ["<= 0.1.3"]
+```
+
+> This is a cross-post of [the official security advisory][ml]. The official post
+> contains a signed version with our PGP key, as well.
+
+# XSS in mdBook's search page
```
Comment (tarcieri) [inline crates/mdbook/RUSTSEC-0000-0000.md:19]: The linter wants the title first:

```suggestion
# XSS in mdBook's search page

> This is a cross-post of [the official security advisory][ml]. The official post
> contains a signed version with our PGP key, as well.
```

Comment (pietroalbini) [reply crates/mdbook/RUSTSEC-0000-0000.md]: Yep! Noticed it as soon as you commented :D

**Comment by pietroalbini** [comment]
Seems like assigning an ID failed?

**Comment by tarcieri** [comment]
Yes, looks like there's a bug in the ID assigner when assigning the first new vulnerability of a given year (happy new year!)

**Comment by tarcieri** [comment]
Assigned `RUSTSEC-2021-0001` in #548

**Pull request merged**